docs/verifybds.sh

```diff
@@ -0,0 +1,38 @@
+ns=$1
+if [[ $ns == "" ]]; then
+    ns="openebs"
+fi 
+cspList=$(kubectl get csp -o jsonpath='{.items[*].metadata.name}')
+bdList=$(kubectl -n $ns get blockdevices -o jsonpath='{.items[*].metadata.name}')
+
+for csp in $cspList
+do
+        echo $csp
+        pod=$(kubectl -n $ns get pods -l openebs.io/cstor-pool=$csp -o jsonpath='{.items[*].metadata.name}')
```

Comment:
Q: What if more than one pod exists? Either it can be in Terminating/Evicted state.